src/css/specific/top-banner.css

```diff
@@ -20,7 +21,8 @@
   color: #087299;
```

Comment:
looking at the design again, it seems that the text should be black (#000) here too. In this case, update this color value and then you don't need the `.announcement p` block from lines 49-51